apps/charterafrica/.env.template

```diff
@@ -32,3 +32,12 @@ PROJECT_ROOT
 SENDGRID_API_KEY
 SENDGRID_FROM_EMAIL
 SENDGRID_FROM_NAME
+
+# Sentry
```

Comment:
Are this part of this PR or the [sentry](https://github.com/CodeForAfrica/ui/pull/446) closed one?